qiita_core/support_files/config_test.cfg

```diff
@@ -23,7 +23,7 @@ LOG_DIR =
 REQUIRE_APPROVAL = True
 
 # Base URL: DO NOT ADD TRAILING SLASH
-BASE_URL = https://localhost:21174
+BASE_URL = https://localhost:8383
```

Comment:
Are there any documents that need to change based on this modification? If I understand correctly, new installations that do not use nginx will have to go to localhost:8383 to use Qiita, is that right?

Reply:
Thanks for the quick review. That is right if they use the exact same configuration that the one that is in the example and being used in Travis; however, in local machines using 80 works fine - at least in my laptop. Now, I checked all the documentation and install instructions and updated the places that needed updating. However, happy to add something else, if you consider it proper; just let me know.

Reply:
I might be missing something - why are we shifting from port 21174 to 8383? It might be good to put a comment in on that.

Reply:
Adding some lines to clarify this; the reason in nginx and the multiple workers ...